index.js

```diff
@@ -43,8 +43,8 @@ exports.Option = Option;
 
 function Option(flags, description) {
   this.flags = flags;
-  this.required = ~flags.indexOf('<');
-  this.optional = ~flags.indexOf('[');
+  this.required = flags.indexOf('<') >= 0;
+  this.optional = flags.indexOf('[') >= 0;
   this.bool = !~flags.indexOf('-no-');
```

Comment:
You can also convert this line as you did with previous one.

Reply:
I can do it if you want, but it will only be for readability/consistency, as the result is boolean as-is.

Reply:
Yes, for consistency purpose.

Reply:
No problem, it's done. :)